example/ios/Podfile

```diff
@@ -15,49 +15,64 @@ def parse_KV_file(file, separator='=')
   if !File.exists? file_abs_path
```

Comment:
Flutter command said the Podfile was out-of-date and suggested regenerating; I didn't manually modify this.